packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -62,11 +62,11 @@
   "org_logout_all_text": "SFDX: Log Out from All Authorized Orgs",
   "org_logout_default_text": "SFDX: Log Out from Default Org",
   "org_open_default_scratch_org_text": "SFDX: Open Default Org",
-  "override_conflicts_on_push_description": "Specifies whether to always use --forceoverwrite when you run force:source:push on save",
+  "ignore_conflicts_on_push_description": "Specifies whether to always use --ignore-conflicts when you run project:deploy:start on save",
   "prefer_deploy_on_save_enabled_description": "Specifies whether to always run deploy instead of push when a local source file is saved and `Push or deploy on save` is enabled.",
   "project_generate_text": "SFDX: Create Project",
   "project_generate_with_manifest_text": "SFDX: Create Project with Manifest",
-  "push_or_deploy_on_save_enabled_description": "Specifies whether or not to automatically run force:source:push (for source-tracked orgs) or force:source:deploy (for non-source-tracked orgs) when a local source file is saved.",
+  "push_or_deploy_on_save_enabled_description": "Specifies whether or not to automatically run project:deploy:start (for source-tracked orgs) or force:source:deploy (for non-source-tracked orgs) when a local source file is saved.",
```

Comment:
Both source-tracked orgs and non-source-tracked orgs should now run `project:deploy:start`, right?